src/depends.py

```diff
@@ -1,10 +1,10 @@
 from dependency_injector import containers, providers
 from sqlalchemy.ext.asyncio import async_sessionmaker, create_async_engine
 
-from .settings import Settings
+from settings import Settings
 
 
 class Container(containers.DeclarativeContainer):
     settings = providers.Singleton(Settings)
-    engine = providers.Singleton(create_async_engine, settings.database_url, future=True, echo=True)
+    engine = providers.Singleton(create_async_engine, Settings.database_url, future=True, echo=True)
```

Comment:
Так работать не будет
```suggestion
    engine = providers.Singleton(create_async_engine, settings.provided.database_url, future=True, echo=True)
```